lib/stdlib/doc/src/maps.xml

```diff
@@ -172,6 +172,53 @@ val1
       </desc>
     </func>
 
+    <func>
+      <name name="intersect" arity="2" since="OTP 24.0"/>
+      <fsummary></fsummary>
+      <desc>
+        <p>Intersects two maps into a single map
+        <c><anno>Map3</anno></c>. If a key exists in both maps, the
+        value in <c><anno>Map1</anno></c> is superseded by the value
+        in <c><anno>Map2</anno></c>.</p>
+        <p>The call fails with a <c>{badmap,Map}</c> exception if
+        <c><anno>Map1</anno></c> or <c><anno>Map2</anno></c> is not a
+        map.</p>
+        <p><em>Example:</em></p>
+        <code type="none">
+> Map1 = #{a => "value_one", b => "value_two"},
+  Map2 = #{a => 1, c => 2},
+  maps:intersect(Map1,Map2).
+#{a => 1}</code>
+      </desc>
+    </func>
+
+    <func>
+      <name name="intersect" arity="3" since="OTP 24.0"/>
+      <fsummary></fsummary>
+      <desc>
+        <p>Intersects two maps into a single map
+        <c><anno>Map3</anno></c>. If a key exists in both maps, the
+        value in <c><anno>Map1</anno></c> is combined with the value
+        in <c><anno>Map2</anno></c> by the
+        <c><anno>Combiner</anno></c> fun. When
+        <c><anno>Combiner</anno></c> is applied the key that exists in
+        both maps is the first parameter, the value from
+        <c><anno>Map1</anno></c> is the second parameter, and the value
+        from <c><anno>Map2</anno></c> is the third parameter.</p>
+        <p>The call fails with a <c>{badmap,Map}</c> exception if
+        <c><anno>Map1</anno></c> or <c><anno>Map2</anno></c> is not a
+        map. The call fails with a <c>badarg</c> exception if
+        <c><anno>Combiner</anno></c> is not a fun that takes three
+        arguments.</p>
+        <p><em>Example:</em></p>
+        <code type="none">
+> Map1 = #{a => "value_one", b => "value_two"},
+  Map2 = #{a => 1, c => 2},
+  maps:intersect(fun(a, V1, V2) -> {V1, V2} end,Map1,Map2).
```

Comment:
I think the example is clearer if you spell the parameters out (following the type spec):
```suggestion
  maps:intersect(fun(_Key, Value1, Value2) -> {Value1, Value2} end, Map1, Map2).
```